components/Container.js

```diff
@@ -16,6 +18,18 @@ const Container = () => {
         </Typography>
       </div>
       <div className="flex ml-3 mt-6 space-x-6 mr-4"></div>
+
+      {/* can be editied to store new cards */}
+      <div className="flex flex-row space-x-5 p-10">
+        <CreateCard
+          spaceName="Create a Space"
+          description="insert some sort of tagline or feature description "
+        />
+        <JoinCard
+          spaceName="Join a Space"
+          description="insert some sort of tagline or feature description "
+        />
```

Comment:
These cards are nearly identical. Can you abstract it into one? You can probably add a background prop (ex `light` or `dark`) and conditionally change it in `makeStyles`